lib/graphql/query/validation_pipeline.rb

```diff
@@ -86,24 +79,23 @@ def ensure_has_validated
           end
 
           if @validation_errors.none?
-            @query_analyzers = build_analyzers(@schema, @max_depth, @max_complexity)
-            # if query_analyzers.any?
-            #   analysis_results = GraphQL::Analysis.analyze_query(@query, query_analyzers)
-            #   @analysis_errors = analysis_results
-            #     .flatten # accept n-dimensional array
-            #     .select { |r| r.is_a?(GraphQL::AnalysisError) }
-            # end
```

Comment:
lol, what the heck was all this commented-out code ... I guess it was replaced by the same stuff you mentioned above 😅